website/static/js/signUp.js

```diff
@@ -103,6 +103,10 @@ var ViewModel = function(submitUrl, campaign) {
     };
 
     self.submit = function() {
+        if (self.submitted()) {
+            $osf.growl('Already submitted', 'You cannot sign up more than once.', 'error');
```

Comment:
The `type` here should be `danger` instead of `error`. Also, the function defaults to `danger` so it's not necessary to pass it in. https://github.com/CenterForOpenScience/osf.io/blob/develop/website/static/js/osfHelpers.js#L26